### PR TITLE
Printing vstest diagnostics when vstest crashes.

### PR DESCRIPTION
**Pull request by amanda-tarafa**
Trying to get some info towards solving #906.

### CODE REVIEW

**Comment by amanda-tarafa** [comment]
@iantalarico , @ILMTitan Please see my last comment on #906. Thanks

**Comment by iantalarico** [comment]
@amanda-tarafa Looks like this test run failed.  That's convenient :)

File: TestScript.ps1 [hunk 1]
```diff
@@ -39,6 +39,7 @@ OpenCover.Console.exe -register:user -target:vstest.console.exe -targetargs:$tes
     -filter:$filter -returntargetcode
 
 if ($LASTEXITCODE) {
+    Get-Content diagnostics.txt | foreach {Write-Host $_}
```
Comment (ILMTitan) [inline TestScript.ps1]: `Write-Host` does accept pipeline input. You could just write `Get-Content diagnostics.txt | Write-Host`.

Comment (amanda-tarafa) [reply TestScript.ps1]: Done

**Pull request merged**